source/reference/config/triggers.txt

```diff
@@ -241,6 +319,34 @@ trigger configuration files:
        | ``object``
      - .. include:: /includes/trigger-project-expression.rst
 
+   * - | ``event_processors.config.account_id``
+       | ``string``
+     - An AWS account ID. For more information on how to find the account ID,
+       see step 1 (Setup the MongoDB Partner Event Source) of the procedure on the 
+       :ref:`Send Trigger Events to AWS EventBridge <aws-eventbridge>` page.
+
+   * - | ``event_processors.config.region``
+       | ``string``
+     - An AWS region.
+
+   * - | ``event_processors.config.extended_json_enabled``
+       | ``boolean``
+     - False by default. If true, entended JSON is enabled.
+       
+       Triggers convert the BSON types in event objects into standard JSON types. 
+       To preserve BSON type information, you can serialize event objects into 
+       :manual:`Extended JSON format </reference/mongodb-extended-json>`instead. 
+       Extended JSON preserves type information at the expense of readability and 
+       interoperability.
+
+   * - | ``error_handler.config.enabled``
+       | ``boolean``
+     - If true, the error handler is enabled. 
```

Comment:
maybe update to specify when/how the error handler is invoked (or link to the new error handling section)

Reply:
linked to ref from other PR, so won't work for now